<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Restarteur</class>
 <widget class="QMainWindow" name="Restarteur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Restarteur</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>541</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>771</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>501</width>
        <height>131</height>
       </rect>
      </property>
      <property name="whatsThis">
       <string>Veuillez mentionner les chemins d'accès pour vos différents serveurs</string>
      </property>
      <property name="title">
       <string>Chemins d'accès </string>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>121</width>
         <height>16</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>Chemin d'accès realmd  :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditChemAcces">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Chemin d'accès world :</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditChemAcces_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BoutonCheminAccesRealmd">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>40</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Parcourir</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BoutonCheminAccesWorld">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>90</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Parcourir</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BoutonLancerServeur1">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>40</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Lancer serveur</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BoutonLancerServeur2">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>90</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Lancer serveur</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRACRealmd">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>40</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Redémarrer après crash</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxRACWorld">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>90</y>
         <width>141</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Redémarrer après crash</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>180</y>
        <width>411</width>
        <height>221</height>
       </rect>
      </property>
      <property name="title">
       <string>Mise a jour</string>
      </property>
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string>Modifiez la date et l'heure de votre mise a jour (il est conseillé de mettre a jour tard dans la nuit , vers 3h ou 4h du matin)</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nouveau chemin d'accès realmd</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BoutonCheminAccesRealmd_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>80</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Parcourir</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditChemAcces_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>120</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Nouveau chemin d'accès world</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BoutonCheminAccesWorld_2">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>140</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Parcourir</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditChemAcces_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BoutonMajServeur2">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>140</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Mettre a jour le world</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BoutonMajServeur1">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>80</y>
         <width>121</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Mettre a jour le realmd</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_9">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>181</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Mettre a jour les deux serveurs</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>30</y>
        <width>211</width>
        <height>471</height>
       </rect>
      </property>
      <property name="title">
       <string>Etat serveur</string>
      </property>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat serveur realmd :</string>
       </property>
      </widget>
      <widget class="QLabel" name="LabelEtatRealmd">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>60</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
      <widget class="QLabel" name="LabelEtatWorld">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>120</y>
         <width>46</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Off</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>101</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Etat serveur world</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <addaction name="actionOuvrir_session"/>
    <addaction name="actionEnregistrer_session"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuOptions"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOuvrir_session">
   <property name="text">
    <string>Ouvrir session</string>
   </property>
  </action>
  <action name="actionEnregistrer_session">
   <property name="text">
    <string>Enregistrer session</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuitter</sender>
   <signal>activated()</signal>
   <receiver>Restarteur</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
